modules/configuring-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,158 @@
+    folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>" <3>
+    cluster: cluster <3>
+    resourcePool: "/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>" <3>
+    diskType: thin
+    failureDomains: <4>
+    - name: <machine_pool_zone_1> <5>
+      region: <region_tag_1> <6>
+      zone: <zone_tag_1> <7>
+      topology: <8>
+        datacenter: <datacenter1> <9>
+        computeCluster: "/<datacenter1>/host/<cluster1>" <10>
+        resourcePool: "/<datacenter1>/host/<cluster1>/Resources/<resourcePool1>" <11>
+        networks: <12>
+        - <VM_Network1_name>
+        datastore: "/<datacenter1>/datastore/<datastore1>" <13>
+    - name: <machine_pool_zone_2>
+      region: <region_tag_2>
+      zone: <zone_tag_2>
+      topology:
+        datacenter: <datacenter2>
+        computeCluster: "/<datacenter2>/host/<cluster2>"
+        networks:
+        - <VM_Network2_name>
+        datastore: "/<datacenter2>/datastore/<datastore2>"
+        resourcePool: "/<datacenter2>/host/<cluster2>/Resources/<resourcePool2>"
+        folder: "/<datacenter2>/vm/<folder2>"
+# ...
+----
+
+<1> You must define set the `TechPreviewNoUpgrade` as the value for this parameter, so that you can use the VMware vSphere region and zone enablement feature.
+<2> An optional parameter for specifying a vCenter cluster. You define a zone by using a tag from the `openshift-zone` tag category. If you do not define this parameter, nodes will be distributed among all defined failure-domains.
+<3> The default vCenter topology. The installation program uses this topology information to deploy the bootstrap node. Additionally, the topology defines the default datastore for vSphere persistent volumes.
+<4> Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a datastore object. A failure domain defines the vCenter location for {product-title} cluster nodes. If you do not define this parameter, the installation programs uses the default vCenter topology.
+<5> Defines the name of the failure domain. Each failure domain is referenced in the `zones` parameter to scope a machine pool to the failure domain.
+<6> You define a region by using a tag from the `openshift-region` tag category. The tag must be attached to the vCenter datacenter.
+<7> You define a zone by using a tag from the `openshift-zone tag` category. The tag must be attached to the vCenter datacenter.
+<8> Specifies the vCenter resources associated with the failure domain.
+<9> An optional parameter for defining the vSphere datacenter that is associated with a failure domain. If you do not define this parameter, the installation programs uses the default vCenter topology.
+<10> An optional parameter for stating the absolute file path for the compute cluster that is associated with the failure domain. If you do not define this parameter, the installation programs uses the default vCenter topology.
```

Comment:
Small nit: 

```suggestion
<10> An optional parameter for stating the absolute file path for the compute cluster that is associated with the failure domain. If you do not define this parameter, the installation programs use the default vCenter topology.
```

Reply:
Thanks, @stevsmit . Updated. Such instances should state "program" and not "programs. The context of "use" is now correct. Please let me know your thoughts.